llms/mlx_lm/LORA.md

```diff
@@ -95,6 +95,28 @@ mlx_lm.generate \
     --prompt "<your_model_prompt>"
 ```
 
+For generation using the API:
```

Comment:
This doesn't look like the completion or chat format we use for LoRA. Shouldn't we follow one of those formats here (and specify it)?